helpers/transformers/web.js

```diff
@@ -0,0 +1,77 @@
+const { tint } = require('./utils');
+const cfonts = require('cfonts');
+const fs = require('fs');
+
+function build(BRAND) {
+	const cwd = `../../brands/${BRAND}`;
+
+	const { COLORS } = require(`${cwd}/tokens/colors`);
+	const { SPACING } = require(`${cwd}/tokens/spacing`);
+	const { LAYOUT } = require(`${cwd}/tokens/layout`);
+	const { TYPE } = require(`${cwd}/tokens/type`);
+
+	// COLORS
+	const tints = {};
+	Object.keys(COLORS).map(color => (tints[color] = { ...tint(COLORS[color]) }));
+
+	// TYPE
+	const brandfonts = {};
+	TYPE.brandfonts.map(font => {
+		if (font.name) {
+			if (!brandfonts['@font-face']) brandfonts['@font-face'] = [];
+
+			brandfonts['@font-face'].push({
+				fontFamily: font.name,
+				src: `url("${font.files.woff2}") format("woff2"), url("${font.files.woff}") format("woff")`,
+				weight: font.weight,
+				style: font.style,
+			});
+		} else if (font.fontFamily) {
+			brandfonts['fontFamily'] = font.fontFamily;
+		}
+	});
+
+	console.log();
+	cfonts.say(`${BRAND} TOKENS`, {
+		font: 'chrome',
+		colors: [COLORS.primary, COLORS.hero, COLORS.borderDark],
+		space: false,
+	});
+
+	const content = {
+		SPACING: SPACING.map(space => `${space / 16}rem`),
+		COLORS: {
+			tints,
+			...COLORS,
+		},
+		LAYOUT,
+		TYPE: {
+			brandfonts,
+		},
+	};
+
+	try {
+		fs.mkdirSync('dist/');
+	} catch (error) {
+		if (error.code !== 'EEXIST') {
+			console.error(error);
+			process.exit(1);
+		}
+	}
+
+	try {
+		fs.writeFileSync('dist/index.js', JSON.stringify(content), { encoding: 'utf8' });
```

Comment:
Isn't this writing some JSON to a JS file without exporting it so importing it would do nothing?

Reply:
I don't understand the question :)
This is writing out the file that is the main entry point for this package... this is run from a `brand/` package so `dist/` is relative to that cwd

Reply:
Yes but it'll write a file that looks like this which wouldn't export the content.

```jsx
{"tokenThing": "some token thing"}
```

Reply:
ahh `module.exports = ` is missing :) good find!